src/force-update-screen/ForceUpdateScreen.tsx

```diff
@@ -0,0 +1,104 @@
+import {ThemeText} from '@atb/components/text';
+import {Button} from '@atb/components/button';
+import {Logo} from '@atb/assets/svg/mono-icons/logo';
+import {Linking, Platform, ScrollView, View} from 'react-native';
+import React, {useState} from 'react';
+import {MessageInfoBox} from '@atb/components/message-info-box';
+import {ANDROID_STORE_ID, IOS_STORE_ID} from '@env';
+import {StyleSheet, useTheme} from '@atb/theme';
+import {StaticColorByType} from '@atb/theme/colors.ts';
+import {ForceUpdateTexts, useTranslation} from '@atb/translations';
+import {ExternalLink} from '@atb/assets/svg/mono-icons/navigation';
+import Bugsnag from '@bugsnag/react-native';
+
+const themeColor: StaticColorByType<'background'> = 'background_accent_0';
+
+export const ForceUpdateScreen = () => {
+  const [error, setError] = useState<boolean>(false);
+  const styles = useStyles();
+  const {theme} = useTheme();
+  const {t} = useTranslation();
+  const iconDimension = 80;
+
+  return (
+    <View style={styles.container}>
+      <View style={styles.mainView}>
+        <ScrollView>
+          <View style={styles.icon}>
+            <Logo
+              width={iconDimension}
+              height={iconDimension}
+              fill={theme.static.background.background_accent_0.text}
+            />
+          </View>
+          <ThemeText
+            type="body__primary--big--bold"
+            style={styles.title}
+            color={themeColor}
+          >
+            {t(ForceUpdateTexts.header)}
+          </ThemeText>
+          <ThemeText style={styles.subText} color={themeColor}>
+            {t(ForceUpdateTexts.subText)}
+          </ThemeText>
+          <Button
+            rightIcon={{svg: ExternalLink}}
+            onPress={() => {
+              const link = Platform.select({
+                ios: `itms-apps://apps.apple.com/us/app/${IOS_STORE_ID}`,
+                android: `market://details?id=${ANDROID_STORE_ID}`,
```

Comment:
These strings should ideally come from Firestore, as they could change by the time the feature is activated. E.g. Apple's url used to be of the format `https://itunes.apple.com/app/id0123456789`.
I think configuration->urls is a good place for it, and then it is also possible to adapt language, using `us` for `eng` and `no` for `nob` in Apple's url.

Reply:
I suggest to keep itms-app since it calls directly the store app, https goes first to a web-browser then opens the app. about the lang, is not need to pass it since the app store opens in the user's default language. (iOS)

Reply:
I don't see the benefit in only keeping `itms-app` locally hard coded.
Regarding no/us, it is more of a nice to have :+1:

Reply:
`itms-apps` is defined as Apple URL Schemes, and they are only linked immediately to the app, it is probably that is not going to change. We might end up adding it to Firestore, but never change it again.

Reply:
I agree that it is unlikely to change, and hopefully it won't. However I'm suggesting to keep the entire string `"itms-apps://apps.apple.com/us/app/${IOS_STORE_ID}"` as value in a url in Firestore, so it is no extra work to also include `itms-apps://`, and it gives flexibility if needed.

Reply:
I will move this to firestore :)